torch/csrc/api/include/torch/serialize.h

```diff
@@ -40,6 +40,32 @@ void save(const Value& value, SaveToArgs&&... args) {
   archive.save_to(std::forward<SaveToArgs>(args)...);
 }
 
+/// Serializes the given `tensor_vec` of type `std::vector<torch::Tensor>`.
+///
+/// To perform the serialization, a `serialize::OutputArchive` is constructed,
+/// and all arguments after the `tensor_vec` are forwarded to its `save_to`
+/// method. For example, you can pass a filename, or an `ostream`.
+///
+/// \rst
+/// .. code-block:: cpp
+///
+///   std::vector<torch::Tensor> tensor_vec = { torch::randn({1, 2}), torch::randn({3, 4}) };
+///   torch::save(tensor_vec, "my_tensor_vec.pt");
+///
+///   std::vector<torch::Tensor> tensor_vec = { torch::randn({5, 6}), torch::randn({7, 8}) };
+///   std::ostringstream stream;
+///   torch::save(tensor_vec, stream);
```

Comment:
This might just be me not understanding the convention. The first time I saw `torch::save(tensor, stream)`, I tried to repeatedly call this method on the same stream to save multiple tensors, and later discovered that this will corrupt the header. Shall we be more explicit in the doc to say that the same stream cannot be used in multiple `save` invocations?

Reply:
Added comment to make this more explicit